cfme/infrastructure/virtual_machines.py

```diff
@@ -453,6 +453,17 @@ def is_displayed(self):
                 self.context['object'].name in [row.name.text for row in self.affected_vms.rows()])
 
 
+class InfraVmOsView(InfraVmView):
+    """The Operating System page"""
+    title = Text('#explorer_title_text')
+    basic_information = SummaryTable(title="Basic Information")
+
+    @property
+    def is_displayed(self):
+        expected_title = '"OS Info" for Virtual Machine "{}"'.format(self.context['object'].name)
```

Comment:
Please use `f string` here

Reply:
I try to use f string and was wondering why it wasn't used here, until I tried to implement it. There are too many quotes. I started down the escape character route, but that makes things real ugly. If you have an easier solution, let me know. The 'object' quotes would need to be inside on double quotes and the entire string is inside of single. We need 3 types of quotes.

Reply:
@prichard77 you could split it into steps 
```python
        name = self.context['object'].name
        expected_title = f'"OS Info" for Virtual Machine "{name}"'
```

Reply:
Sure, but is this really better than the old format?

Reply:
I went ahead and updated to f' .